en-US/firefox/products/lockwise.lang

```diff
@@ -0,0 +1,82 @@
+## NOTE: https://bedrock-demo-lockwise.oregon-b.moz.works/firefox/lockwise/
+## URL: https://bedrock-demo-lockwise.oregon-b.moz.works/%LOCALE%/firefox/lockwise/
+
+
+# HTML page title
+;Firefox Lockwise — password manager — take your passwords everywhere
+Firefox Lockwise — password manager — take your passwords everywhere
+
+
+# Page discription
+;Firefox Lockwise lets you securely access the passwords you’ve saved in Firefox from anywhere— even outside of the browser. Features 256-bit encryption and Face/Touch ID.
+Firefox Lockwise lets you securely access the passwords you’ve saved in Firefox from anywhere— even outside of the browser. Features 256-bit encryption and Face/Touch ID.
+
+
+# Main header
+;Take your passwords everywhere
+Take your passwords everywhere
+
+
+# Discription under header
+;Securely access the passwords you’ve saved in Firefox from anywhere — even outside of the browser.
+Securely access the passwords you’ve saved in Firefox from anywhere — even outside of the browser.
+
+
+# iTunes download button
+;Download on the App Store
+Download on the App Store
+
+
+# Google play download button
+;GET IT ON Google Play
```

Comment:
Is there are reason why `GET IT ON` is in all caps?

Reply:
This was how is was in the copy given to me. I can change it  if that doesn't make sense?

Reply:
Could you please ask? We have raised questions to copies that went through many rounds of approvals and were changed in the end.

Reply:
I just checked and it looks like GET IT ON being in caps is part of the google brand standards for the google play button.

Reply:
I don't know how that works with other languages though: https://play.google.com/intl/en_us/badges/

Reply:
If this is a button with the Google download image, then it may not be an issue. It just looks weird  this way.  We have localized this string before: https://transvision.mozfr.org/?recherche=get+it+on+google+play&repo=mozilla_org&sourcelocale=en-US&locale=de&search_type=strings_entities